### DISCUSSION TITLE
Samsung wind-free AR12BSEAMWKX communication protocol

### DISCUSSION THREAD

**Discussion by FelixFD** [omerfaruk-aran/esphome_samsung_hvac_bus#119] [Questions]
I recently got a Samsung windfree 12000BTU aircon (AR12BSEAMWKX) installed and I am looking to connect it to this project. However looking at the communication terminals both on the indoor and outdoor unit only 3 wires are used. As seen in the picture. 
![image](https://github.com/lanwin/esphome_samsung_ac/assets/52816116/7c80e666-5e4f-45e8-9a17-58959e83e352) 
I was under the impression that all new models use the NASA protocol with 2 wires - F1,F2 however i could be wrong. 

Is there solution to get the F1 , F2 wires through an adaptor card or something similar?

**Comment by lanwin** [comment]
Hi @FelixFD that looks strange. My AC has the same ports but only for power. And additional ports for the communication. 

Can you please contact your seller and ask how this devices does communication? 

Maybe this device did not control anything. Maybe the indoor unit can only powers on/off the outdoor device. But I am not a AC expert so this are only guesses.

**Reply by FelixFD** [reply]
Thanks for the response, the unit is not a fixed speed so I think there must be some sort of communication between the two units. I have already contacted the seller just waiting for a response.

**Reply by FelixFD** [reply]
Update:

I have heard back from the seller.

The model I have installed (AR12BSEAMWKX) does not offer NASA or any other forms of wired communication - (besides the single core between the indoor and outdoor unit.) 

However, the model AR12BSAAAWK/FA Does implement the NASA protocol.

I think it's time we start to create a compatibility list.

**Reply by lanwin** [reply]
Sad to here. I add a compat List to the readme.
